platform/broadcom/docker-syncd-brcm-dnx-rpc/Dockerfile.j2

```diff
@@ -1,5 +1,7 @@
 {% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
-FROM docker-syncd-brcm-dnx-{{DOCKER_USERNAME}}:{{DOCKER_USERTAG}}
+ARG BASE=docker-syncd-brcm-dnx-{{DOCKER_USERNAME}}:{{DOCKER_USERTAG}}
+
+FROM $BASE as base
```

Comment:
Why dockers/**/Dockerfile is using 'FROM AS', platform/**/Dockerfile is using 'FROM as'?

Reply:
Functionally speaking, there's no difference. I think it was just a copy-paste difference. However, I _think_ newer versions of Docker will print a warning when seeing lowercase `as`, but won't fail the build.

Reply:
Fixed case.